mypy.ini

```diff
@@ -1,36 +1,33 @@
 # Regular configuration file (can be used as base in other projects, runs in CI)
 
 [mypy]
-# Modified in `tests.yml`
-incremental = true
```

Comment:
Btw, I think that we need this to check both modes.

Reply:
Can you explain? It's enabled by default, so having it in the config like this makes no difference AFAICT. We use `--no-incremental` to disable it when running mypy on the stubs.

Reply:
The comment itself helps, so I would prefer to keep this one :)
This setting also caused a lot of trouble in the past.